content/Cookies/Cookies.q.md

```diff
@@ -0,0 +1,947 @@
+
+/// type=MS, answer=[3,4]
+
+Which statements are true about a cookie?
+
+ - It accesses a cookie value.
+
+ - It creates and deletes cookies in PHP.
+
+ - It is a small amount of information stored on the user's computer.
+
+ - It is used for application state tracking and identification purposes. 
+
+ - It sends a URL encoded cookie together with the rest of the HTTP headers.
+
+
+/// type=MS, answer=[2,5]
+
+Which statements correctly describe the `setcookie()` function?
+
+ - It accesses a cookie value.
+
+ - It creates and deletes cookies in PHP.
+
+ - It is a small amount of information stored on the user's computer.
+
+ - It is used for application state tracking and identification purposes. 
+
+ - It sends a URL encoded cookie together with the rest of the HTTP headers.
+
+
+/// type=MS, answer=[2,3,4,5]
+
+Which of the following are `setcookie()` parameters?
+
+ - `URL`
+
+ - `name`
+
+ - `value`
+
+ - `domain`
+
+ - `expires`
+
+
+/// type=SS, answer=[1]
+
+Which statement best describes the `$_COOKIE` superglobal variable.
+
+ - It is used to access a cookie value.
+
+ - It creates and deletes cookies in PHP.
+
+ - It is a small amount of information stored on the user's computer.
+
+ - It is used for application state tracking and identification purposes. 
+
+ - It sends a URL encoded cookie together with the rest of the HTTP headers.
+
+
++++
+
+
++++
+
+### Part 3: Finding and Fixing Errors
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+setcookie("user", "My Cookie" time() + 60);
+
+if (isset($_COOKIE['user'])) {
+	echo "The cookie value is: " . $_COOKIE['user'];
+}
+```
+/// type=SS, answer=[2]
+
+Execute the program. What is its output?
+
+ - It prints `user`.
+
+ - It produces an error.
+
+ - It prints `My Cookie`.
+
+ - No output is displayed.
+
+ - It prints `The cookie value is: My Cookie`.
+
+
+/// type=SS, answer=[4]
+
+What is the error message?
+
+ - Fatal error: Cannot use `[]` for reading on line number 15
+
+ - Warning: Cookie names must not be empty on line number 6
+
+ - syntax error, unexpected `'60'` (T_LNUMBER) on line number 4
+
+ - syntax error, unexpected `'time'` (T_STRING), expecting `','` or `')'` on line number 2
+
+ - syntax error, unexpected `''user''` (T_CONSTANT_ENCAPSED_STRING), expecting `','` or `';'` on line number 5
+
+
+/// type=MS, answer=[4,5]
+
+Which statements correctly describe the error?
+
+ - There is no open parenthesis `(` after `setcookie` on line 2.
+
+ - There is no semicolon `;` at the end of the statement on line 2.
+
+ - There is an addition operator `+` specified between `time()` and `60` on line 2.
+
+ - There is no comma `,` specified between `"My Cookie"` and `time() + 60` on line 2.
+
+ - On line 2, the statement `setcookie("user", "My Cookie" time() + 60);` is invalid.
+
+:::
+
+
+/// type=CR, answer=[tests/Cookies/MissingCommaBetweenArgumentTest.php]
+
+Correct the code so that it outputs the string `The cookie value is: My Cookie`.
```

Comment:
The string 'The cookie value is: My Cookie`.' will only be displayed when the page is reloaded after its execution.

Will this work?
"Correct the code so that it outputs the string `The cookie value is: My cookie` when the page is reloaded after its execution."

I am not sure with the term `reloaded` though since there is no reload in the codestop editor.